tests/external_board_native/external_boards/native/Makefile

```diff
@@ -0,0 +1,6 @@
+# This must be a different name than 'board' as it is implemented by 'native'
+MODULE = board_external_native
```

Comment:
True for this specific case but should still be `MODULE = board` when porting a custom board